bears/js/ESLintBear.py

```diff
@@ -35,7 +35,11 @@ def create_arguments(filename, file, config_file,
         """
         :param eslint_config: The location of the .eslintrc config file.
         """
-        args = '--no-ignore', '--no-color', '-f=json', '--stdin'
+        args = '--no-ignore',\
+               '--no-color',\
+               '-f=json',\
+               '--stdin',\
+               '--stdin-filename=' + filename
```

Comment:
Backslashes 😧 

I wonder why this isn't checked by static analysis checks, using backslashes to escape newlines is considered a pretty bad practice. However!

1. `args = 1, 2, 3` is just syntactic sugar for `args = (1, 2, 3)` (which is creating a `tuple` and is therefore syntactic sugar for `args = tuple(1, 2, 3)`)
2. Newlines don't need to be escaped inside parentheses.

Which means that a multiline
```
args = 1,\
       2,\
       3
```
can also be written as
```
args = (
    1,
    2,
    3,
)
```